corehq/apps/app_manager/tests/test_views.py

```diff
@@ -88,3 +99,92 @@ def test_edit_commcare_profile(self):
         custom_properties = content["changed"]["custom_properties"]
 
         self.assertEqual(custom_properties["random"], "changed")
+
+    def _load_app(self, filename):
```

Comment:
These utils methods are available in [`corehq.apps.app_manager.tests.util.TestFileMixin`](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/tests/util.py#L39) You could subclass `TestFileMixin` and reuse those same methods.